requirements/AOCS-GNC_AURORA_0001/taste_project/Default_Deployment.aadl

```diff
@@ -0,0 +1,130 @@
+PACKAGE deploymentview::DV::Node_Linux
```

Comment:
This file is automatically generated - please remove

Reply:
Done